sorting_algos/quicksort.py

```diff
@@ -1,62 +1,26 @@
-def quickSort(arr):
-    less = []
-    pivotList = []
-    more = []
-    if len(arr) <= 1:
-        return arr
+"""
+Write a function that accepts an array of unsorted integers,
+and returns a sorted array by a recursive quicksort algorithm.
+"""
+
+
+def quicksort(my_list):
+    """Define a recursive quicksort function."""
+    left = []
+    pivot_list = []
+    right = []
+    if len(my_list) < 2:
+        return my_list
     else:
```

Comment:
Since you are returning in the `if` statement, you do not need to put the rest of the code in an `else` statement.